x509/src/certificate.rs

```diff
@@ -25,6 +28,57 @@ pub fn default_zero() -> u32 {
 /// Version  ::=  INTEGER  {  v1(0), v2(1), v3(2)  }
 pub const X509_CERT_VERSION: u8 = 2;
 
+/// Version identifier for X.509 certificates. In practice, only v3 is used.
+#[derive(Clone, Debug, Copy, PartialEq, Eq, PartialOrd)]
+pub enum X509Version {
```

Comment:
In the spirit of [RFC356](https://github.com/rust-lang/rfcs/blob/master/text/0356-no-module-prefixes.md) I'd suggest simply calling this `Version`.

That also follows the convention used by other version types in this repo.

Reply:
ok. will change as i mop up cargo hack findings.

Reply:
#410 allows you to derive this type.